shared/fs/index.js

```diff
@@ -156,4 +145,16 @@ const stylesRowRightBox = {
   paddingRight: globalMargins.small,
 }
 
+const iconPlaceholderIconStyle = {
+  marginRight: globalMargins.small,
+  fontSize: 32,
+}
+
+const rowTextStyles = memoize(color => ({
+  color,
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
```

Comment:
this and whitespace might not work in rn, just a heads up

Reply:
Oh hmm what would a good alternative be?

Reply:
i don't remember offhand, you can look around in the code but likely you want to use isMobile ? {} : {}

Reply:
Got it thanks!